x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/translations.ts

```diff
@@ -81,3 +81,11 @@ export const EXCEPTION_BUILDER_INFO = i18n.translate(
     defaultMessage: "Alerts are generated when the rule's conditions are met, except when:",
   }
 );
+
+export const ADD_EXCEPTION_SEQUENCE_WARNING = i18n.translate(
+  'xpack.securitySolution.exceptions.addException.sequenceWarning',
+  {
+    defaultMessage:
+      'This rule is a sequence statement. The exception created will apply to all events in the sequence.',
```

Comment:
@paulewing Do you have any preferences on the language here? I'm thinking that the first sentence could be more specific and say something like this:
```
This rule's query contains an EQL sequence statement.
```